index.js

```diff
@@ -1049,6 +1105,56 @@ class SDK extends EventEmitter {
     return { rawJSON: this._flatten(rawJSON), metadata }
   }
 
+  /**
+   * sync.
+   *
+   * @description Manually sync your hyperdrive
+   * @async
+   * @public
+   * @param {String|BUffer} key - drives key
+   * @param {String} moduleDir - path to module directory
+   * @param {Object} [opts] - dft options
+   * @returns {Array<Object>} - diff array, eg: `change: 'mod', type: 'file', path: '/hello.txt'}`
+   */
+  async sync (key, moduleDir, opts = {}) {
+    const dkey = DatEncoding.encode(
+      crypto.discoveryKey(DatEncoding.decode(key))
+    )
+    let drive = this.drives.get(dkey)
+    if (!drive) {
+      // dat open
+      drive = dat.open(this.store, DatEncoding.decode(key), {
+        sparse: true,
+        sparseMetadata: true
+      })
+
+      await drive.ready()
+    }
+    if (!drive.writable) {
+      this._log('drive is not writable -  can not sync')
```

Comment:
```suggestion
      this._log('drive is not writable -  nothing to sync')
```

This makes it clearer that it's not an error at all since nothing is to be done